src/libraries/System.IO.Pipelines/src/System.IO.Pipelines.csproj

```
@@ -9,8 +9,6 @@ Commonly Used Types:
 System.IO.Pipelines.Pipe
 System.IO.Pipelines.PipeWriter
 System.IO.Pipelines.PipeReader</PackageDescription>
```

revert the branch specific changes:
```suggestion
System.IO.Pipelines.PipeReader</PackageDescription>
    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
    <ServicingVersion>2</ServicingVersion>
```